daemon/manager/manager.go

```diff
@@ -101,17 +101,23 @@ func Execute(configs *options.BootstrapConfigs) error {
 	log.Info("command Start cni successful")
 	defer cniInstaller.Stop()
 
-	setupCloseHandler()
+	*configs.Status = setupCloseHandler()
+	log.Printf("bpfLoader.Restart %v", *configs.Status)
 	return nil
 }
 
-func setupCloseHandler() {
+func setupCloseHandler() int {
 	ch := make(chan os.Signal, 1)
 	signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT, syscall.SIGHUP, syscall.SIGABRT, syscall.SIGTSTP)
 
 	<-ch
```

Comment:
just get daemonset info when kmesh stop:
if we can get, it is restart, do not clean bpf map
if we get err, it is stop, clean bpf map

Reply:
> just get daemonset info when kmesh stop: if we can get, it is restart, do not clean bpf map if we get err, it is stop, clean bpf map

I seems to know your intention. `setupCloseHandler()` is the last step of `Execute()`. It will return until it get signal. Then your get Kmesh restart status, it will be used in `bpf.Stop()` which is defined in defer function of `Execute()`.

But the name of `setupCloseHandler()` is confusing. It looks like it will return immediately after setting the signal handler.